python/cudf/cudf/tests/test_rolling.py

```diff
@@ -122,11 +108,19 @@ def test_rolling_with_offset(agg):
         ],
     )
     gsr = cudf.from_pandas(psr)
-    assert_eq(
-        getattr(psr.rolling("2s"), agg)().fillna(-1),
-        getattr(gsr.rolling("2s"), agg)().fillna(-1),
-        check_dtype=False,
-    )
+    try:
+        assert_eq(
+            getattr(psr.rolling("2s"), agg)().fillna(-1),
+            getattr(gsr.rolling("2s"), agg)().fillna(-1),
+            check_dtype=False,
+        )
+    except AssertionError as e:
+        if agg == "count": 
+            # count for offset-columns does not follow similar conventions
+            # to non-offset-columns.
+            pytest.xfail(reason="Differs from pandas behaviour here") 
+        else:
+            raise e
```

Comment:
Could we mark `"count"` parameter similar to this: https://github.com/rapidsai/cudf/blob/branch-0.16/python/cudf/cudf/tests/test_dataframe.py#L2092-#L2097

```python
pytest.param(
            "count",
            marks=pytest.mark.xfail(
                reason="Differs from pandas behaviour here"
            ),
        )
```

Reply:
Ah, right you are. Apologies, I was following the convention in `test_rolling.py`. I'll switch.

Reply:
Thanks for the suggestion, @galipremsagar. This is cleaner.